### ISSUE TITLE
Cannot update to insights-agent 2.16.1

### ISSUE THREAD

**Issue by souleb** [reactiveops/charts#1153]
### What happened?

I tried to update to version 2.16.1 and I get the following error

```yaml
Helm install failed: error while running post render on files: map[string]interface {}(nil): yaml: unmarshal errors:
  line 35: mapping key "volumes" already defined at line 33
```

It seems there is a duplicate field in the chart.

I use flux to install the chart.

### What did you expect to happen?

I was expecting the new version to update successfully.

### How can we reproduce this?

I encounter this with flux trying to reconcile a new version.

### Version

2.16.1

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by rbren** [comment]
Hi @souleb--sorry you're having touble.

Are you able to share your values.yaml (you should redact the base64token if it's hard-coded). The output of `helm template --debug` would probably help too

**Comment by souleb** [comment]
hello, here are the values

```yaml
    Fleet Install:  true
    Goldilocks:
      Controller:
        Resources:
          Requests:
            Memory:  32Mi
      Enabled:       true
    Insights:
      Cluster:            xxx
      Organization:       xxx
      Token Secret Name:  xxx
    Kube - Bench:
      Enabled:  true
    Kube - Hunter:
      Enabled:  true
    Nova:
      Enabled:  true
    Pluto:
      Enabled:  true
    Polaris:
      Enabled:  true
    Prometheus - Metrics:
      Enabled:                    true
      Install Prometheus Server:  true
    Rbac - Reporter:
      Enabled:  true
    Trivy:
      Enabled:  true
```

**Comment by transient1** [comment]
@souleb Apologies for the delay. This was a tricky one to debug. I was able to reproduce it in a test cluster with FluxCD installed. It was an error in our chart templating, but it only appears with FluxCD (it's related to their post-render mechanism). We have resolved the issue and released chart version 2.17.3 of the insights-agent. I was able to successfully install it in my test cluster. Please give that a try.

**Comment by sudermanjr** [comment]
This has been resolved in the latest version of the chart.
